samples/rqd/blender/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Builds on the latest base image of RQD from Docker Hub
-FROM opencue/rqd
+FROM opencue/rqd2
```

Comment:
This line's change looks like it's from local testing, I think we should revert it?

Reply:
My bad, that one had slipped through with the last commit.
Removed in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1319/commits/ef0f978d36a92a5030a0a497ffa4b7cf3332dc9b